services/graylog/scripts/alerts.template.yaml

```diff
@@ -271,12 +271,12 @@
     grace_period_ms: 0
     backlog_size:
   alert: true
-- title: "${MACHINE_FQDN}: Potential hanging dy-sidecar service detected. Please investigate."
-  description: "${MACHINE_FQDN}: Potential hanging dy-sidecar service detected. Please investigate."
+- title: "${MACHINE_FQDN}: Potential hanging dy-sidecar service detected."
+  description: "${MACHINE_FQDN}: Potential hanging dy-sidecar service detected. Human intervention required. Please investigate."
   priority: 3
   config:
     query: >
-      "unexpectedly failed [OEC" AND container_name:/.*director-v2.*/
+      "not contact dynamic-sidecar to save service state or output ports" AND container_name:/.*director-v2.*/
```

Comment:
NOTE that it is very easy get error message out of sync (we are constantly changing those to make them more "human readable"). I would use instead some sort of identifier code, similar  to the ``OEC`` that you have just removed.

Some suggestions:
1. define some unique "graylog markers" in osparc-simcore 
2. append them in logs from errors you want to detect 

If you like we can create a mechanism that does that automatically. @matusdrobuliak66 might be working in improving the loggers so we could add it. Just keep it in mind.

Reply:
You are correct, and a system for synchronizing this would be nice :) But conceptually this amounts to synchronizingv the ops-repo and the simcore repo, and this synchronization will have to work across multiple deployments and code-versions (releases). So to have this robust this is a big task. Ideally, it would be a system as you propose though

Reply:
small suggestion:
- why not define a unique ID (like this OEC I think is), and create a OR case. This way you ensure you still get the text version (which has all the issues related here by @pcrespov) and we slowly move to the ID based one. This way I think you would cover all the deployments and in a few releases you can remove the text-based one?

Reply:
So I like this unique ID to tag to these errors that require monitoring so that messages can be caught reliably in Graylog.

Also not to be a party buster, I found out that this message is also not that reliable. It can get triggered multiple times due to retries and in the end the sidecar is removed successfully. 

My [upcoming PR](https://github.com/ITISFoundation/osparc-simcore/pull/4040) will provide one that only shows one when a sidecar is detected as waiting for manual user intervention.

Reply:
FYI: updated PR to reflect changes from https://github.com/ITISFoundation/osparc-simcore/pull/4040/files#diff-580b3ba278d2b6f71ef7fa38a284a0b5a5ef7cbf506f8ba7a77990cb22238da0R40-R42